### PR TITLE
Bump version to 1.0.1

### PR DESCRIPTION
**Pull request by kevinlind**
## Description

Bump version to 1.0.1

- Sets AEPCore dependency to v3.5.0
- Sets AEPEdge dependency to v1.4.0
- Uses AEPCore staging in Podfile as AEPCore 3.5.0 is not released yet.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: AEPEdgeConsent.podspec [hunk 1]
```diff
@@ -15,8 +15,8 @@ Pod::Spec.new do |s|
   s.swift_version = '5.1'
 
   s.pod_target_xcconfig = { 'BUILD_LIBRARY_FOR_DISTRIBUTION' => 'YES' }
-  s.dependency 'AEPCore', '>= 3.1.0'
```
Comment (addb) [inline AEPEdgeConsent.podspec:18]: I see the current latest version to be `3.4.2`. Is there a `3.5.0` in pipeline?

Comment (kevinlind) [reply AEPEdgeConsent.podspec:18]: Yes. It includes the added APIs in Date+Format which Consent uses.

File: AEPEdgeConsent.podspec [hunk 2]
```diff
@@ -15,8 +15,8 @@ Pod::Spec.new do |s|
...
-  s.dependency 'AEPEdge', '>= 1.1.0'
+  s.dependency 'AEPCore', '>= 3.5.0'
+  s.dependency 'AEPEdge', '>= 1.4.0'
```
Comment (emdobrin) [inline AEPEdgeConsent.podspec:19]: This one is not a requirement, but are you bumping just for format consistency?

Comment (kevinlind) [reply AEPEdgeConsent.podspec:19]: Yes, for the format consistency.

**Review by emdobrin — APPROVED** [review]

**Pull request merged**